tests/core/fs/storage/test_user_storage.py

```diff
@@ -100,10 +103,12 @@ async def test_vacuum(alice_user_storage):
 
 @pytest.mark.trio
 @customize_fixtures(real_data_storage=True)
-async def test_storage_file_tree(tmp_path, alice):
+async def test_storage_file_tree(tmp_path: Path, alice: LocalDevice):
     manifest_sqlite_db = tmp_path / alice.slug / "user_data-v1.sqlite"
 
+    # Pristine start: DB is not created on FS...
+    assert not manifest_sqlite_db.exists()
     async with UserStorage.run(tmp_path, alice) as aus:
-        assert aus.manifest_storage.path == manifest_sqlite_db
-
-    assert manifest_sqlite_db.is_file()
+        _ = aus.device
+        # ...and now it is !
+        assert manifest_sqlite_db.is_file(), "database should be a file"
```

Comment:
no need for the message in the test assert

Reply:
done

Reply:
done